src/main/java/uk/gov/ch/controller/update/trusts/TrustLinksController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.ch.controller.update.trusts;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.TrustDataCountNotFoundException;
+import uk.gov.ch.model.update.trusts.TrustLinkData;
+import uk.gov.ch.service.update.trusts.TrustLinksService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+@RestController
+public class TrustLinksController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            OracleQueryApplication.APPLICATION_NAME_SPACE);
+    @Autowired
+    private TrustLinksService individualTrusteesService;
+
+    @GetMapping("overseas-entity/{oeNumber}/trusts/beneficial_owners/links")
```

Comment:
This is a different endpoint to the one described in the readme?

Reply:
Yeah I've just fixed the README.md file

The endpoint was changed by Andre further up the PR and I forgot to change it in the README.md